### DISCUSSION TITLE
Looking to hire an experienced Saleor developer

### DISCUSSION THREAD

**Discussion by CopperMS** [saleor/saleor#8989] [Q&A]
Hello all, I've got a website built with Saleor v2.11, and the original developer has been hired by a startup & can't 'moonlight' on other projects. So, I'm looking for an experienced lead developer to handle some minor updates and site changes for the next several months...no major overhauls. I could consider upgrading from v2.11 to 3.x, but a brief review of what's & why's is appropriate first.

Please reply here if interested and I'll relay my private contact info to you.

**Comment by Diego17ccs** [comment]
hello my name is Diego Soules and I am writing to you on behalf of VenSoft, C.A. I wrote to you because I saw your request to hire someone with experience in Saleor 2.11 and as the CEO founder of VenSoft, C. A. I can provide you with the knowledge and team to complete the request or task that you need, feel free to contact me, if you have any questions do not hesitate to contact me, I am at your disposal and to serve you in whatever you need.

**Comment by Rasmez** [comment]
Hi, I'm interested to handle your website, I have been using Saleor a lot for a personal project of mine, so I'm confident that I'm able to solve almost any eventuality with it (at least with a v2.X instance like yours).  ̶I̶'̶l̶l̶ ̶s̶e̶n̶d̶ ̶y̶o̶u̶ ̶a̶ ̶P̶M̶ ̶w̶i̶t̶h̶ ̶m̶y̶ ̶c̶o̶n̶t̶a̶c̶t̶ ̶i̶n̶f̶o̶r̶m̶a̶t̶i̶o̶n̶  Nevermind, Github deactivated the private messaging feature and the reply e-mails (<user>@users.github.com) a long time ago and I didn't notice.

**Comment by CopperMS** [comment]
Hi Ricardo,
 Sorry for the late reply. Do you have a CV/resume or portfolio journal,
along with any references to contact companies/people for past Saleor jobs
you worked on?

Best,
Chad

**Reply by Rasmez** [reply]
Unfortunately I don't have any past work with any companies or people using Saleor specifically available to share because of NDAs and such, but if you want, I can deploy a test here in Github (sans the code covered in the NDAs) to prove my skills and experience.  Sorry for the late response I had a problem with my ISP... It's city-wide problem with the fiber of the only ISP so... not much to do there.

**Comment by CopperMS** [comment]
Hi Diego,
 Sorry for the late reply. Do you have a CV/resume or portfolio journal,
along with any references to contact companies/people for past Saleor jobs
you worked on?

Best,
Chad
